screens/modals/QuantityModal.js

```diff
@@ -68,6 +69,13 @@ export default class QuantityModal extends React.Component {
     const currentQuantityInt = parseInt(this.state.currentQuantity, 10);
     const priceDifference = (currentQuantityInt - initialQuantity) * this.state.product.customerCost;
     this.props.callback(this.state.product, currentQuantityInt, priceDifference);
+    Analytics.logEvent('ConfirmQuantity', {
+      name: 'Apply product quantity',
+      function: 'handleUpdateCart',
+      screen: 'QuantityModal',
```

Comment:
It might be confusing to have `screen` as an event property since `screen_name` is captured with the Analytics too. Perhaps `component`?

Reply:
Good idea! Thanks Tommy